framework/test/mocha/integration/transactions/exceptions/multisignature.js

```diff
@@ -0,0 +1,200 @@
+const expect = require('chai').expect;
```

Comment:
```suggestion
const { expect } = require('chai');
```
Also, header required